test/it/api_auth_it.js

```diff
@@ -588,6 +590,35 @@ describe('Application.authenticateApiRequest',function(){
       assert.equal(scopeFactoryArgs[1][1],requestedScopes[1]);
     });
 
+    it('should call the scope factory with the requested scope when scopeFactory is given a callback', function(done) {
```

Comment:
Instead of these long descriptions try to only have the expectation in the `it` and use `describe` to describe the context, like this:

```javascript
describe('when scopeFactory is given a callback', function() {
  it('should call the scope factory with the requested scope');
});
```

What do you think?